README.md

```diff
@@ -94,9 +94,9 @@ const newman = require('newman'); // require newman in your project
 newman.run({
     collection: require('./sample-collection.json'),
     reporters: 'cli'
-}, function (err) {
+}, function (err, data) {
 	if (err) { throw err; }
-    console.log('collection run complete!');
+    console.log(`collection run complete! Summary Run Data: ${data}`);
```

Comment:
the second argument `data` is an object and using it with string template literal will convert it to `[object Object]`.
So, here it will log: `collection run complete! Summary Run Data: [object Object]` 😕

Reply:
@codenirvana, resolved.  I also updated the documentation for running newman as a library.